### DISCUSSION TITLE
Powrót do moodla

### DISCUSSION THREAD

**Discussion by baktrius** [baktrius/MIMUW_PW_2nd_lab_assignment#1] [Announcements]
Część z was sprzeciwiła się prowadzenia dyskusji przez GitHub'a więc wracamy na Moodle. Odpowiednie forum jest już udostępnione.

**Comment by adespawn** [comment]
O ile forum na moodlu widzę, to nie mogę stworzyć tam nowej wiadomości. @baktrius

**Reply by baktrius** [reply]
Hmmm, coś tam kliknąłem, ale z mojej pozycji ciężko to przetestować. Czy już działa?

**Reply by adespawn** [reply]
Ciągle nie mogę dodać wiadomości. Może problemem jest tryb forum:
```
To jest forum 'Pytań i odpowiedzi'. Aby zobaczyć odpowiedzi innych osób, musisz najpierw sam udzielić odpowiedzi.
```

**Reply by baktrius** [reply]
Dzięki za zwrócenie uwagi. Zmieniłem tryb forum na taki jak był przy zadaniu z Javy. Czy teraz działa?
